spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -671,22 +672,22 @@ it is still possible to dynamically remove metrics from metric registries at run
 === Metric Registries
 
 The `MetricRegistry` is used to maintain a collection of metrics along with their <<pgm-metadata,metadata>>.
-There is one shared singleton of the `MetricRegistry` per scope (_application_, _base_, and _vendor_).
-When metrics are registered using annotations, the metrics are registered in the _application_ `MetricRegistry` (and thus the _application_ scope).
+There is one shared singleton of the `MetricRegistry` per scope (_application_, _base_, _vendor_ and user-defined scopes).
+When metrics are registered using annotations, the metrics are registered in the _application_ `MetricRegistry` (and thus the _application_ scope) by default if the the annotation's _scope_ parameter does not supply a user-defined scope.
 
-When injected, the `@RegistryType` is used as a qualifier to selectively inject either the `APPLICATION`, `BASE`, or `VENDOR` registry.
-If no qualifier is used, the default `MetricRegistry` returned is the `APPLICATION` registry.
+When injected, the `@RegistryScope` is used to selectively inject either an `application`, `base`, `vendor` or a user-defined scope.
+If _scope_ parameter is used, the default `MetricRegistry` returned is the `application` registry.
```

Comment:
"if the _scope_ parameter is used," -> "If the _scope_ parameter is omitted,"

Reply:
Fixed through rebase